src/store/index.js

```diff
@@ -125,7 +128,7 @@ const store = new Vuex.Store({
       // Ideally here we would run all queries in parallel
       // however running through an SSH tunnel doesn't work
       // it only supports one query at a time.
-      const onlyTables = await context.state.connection.listTables()
+      const onlyTables = await context.state.connection.listTables({schema: null})
```

Comment:
This is the big one. Where do we store and expose which schemata we want to view? IMO shouldn't be done before making a connection, so maybe as a top level component on the Table list? 

So first time you connect you see: 
```
> Schemas (3/3) 
  > public 
    > bar_table
    > foo_view
  > x_schema
    > x_table
  > y_schema
    > y_table
```

But then an action on `Schemas` would allow you to filter what is pulled

Reply:
so several things (eg the auto-complete prompt) require having all table and column data on connect. So that is a design limitation. I think pulling all schemas & tables at once is fine.

I'll test your PR later to see what undocumented assumptions it breaks :-)

Reply:
I dislike the datagrip and dbeaver approach to having a 'schema' folder, of course they're schemas, what else are they?

I think it probably needs to show the schemas collapsed initally, even if it pulls all the table information up-front

Reply:
Yeah definitely agree on collapsed. Hadn't thought about the connection problem – been a while since I strayed from postgres ;) 

Yeah, the schema folder is super annoying, though IntelliJ only does it if you enable additional object types / have a wide context. They also allow you to specify a regex-like filter like the following (notice the tree / UI just updates the "Schema Pattern" field)

![Screen Shot 2020-05-07 at 9 46 16 AM](https://user-images.githubusercontent.com/160142/81315549-ac89a100-9047-11ea-819e-beb98048dba0.png)

Maybe we offer just a box like that and leave it to the (likely technical) users to hone as needed? Could default to `*`, or even a DBMS-specific default like `public.*` or `[^postgres].*` for postgres?